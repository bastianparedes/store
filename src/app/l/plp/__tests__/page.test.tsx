import { render } from '@testing-library/react';

import mockDrizzle from '../../../../../__mocks__/drizzle';
import Page from '../page';

jest.mock('../../../../../lib/drizzle', () => mockDrizzle);

describe('Page', () => {
  it('should render', async () => {
    const { container } = render(await Page());
    expect(container).toMatchSnapshot();
  });
});
