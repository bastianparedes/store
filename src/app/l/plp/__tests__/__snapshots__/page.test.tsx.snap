// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page should render 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      Cat√°logo
    </h1>
    <div
      class="productList"
    >
      <div
        class="productItem"
      >
        <a
          href="/l/pdp/1"
        >
          <img
            alt="name1"
            src="/api/image/1"
          />
        </a>
        <p>
          Nombre: 
          name1
        </p>
        <p>
          Precio: $
          1000
        </p>
        <p>
          Cantidad disponible: 
          1
        </p>
      </div>
    </div>
  </div>
</div>
`;
